src/controllers/athlete-controller.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const boom = require('boom')
+const util = require('../util')
+const Athlete = require('../models/athlete')
+
+exports.getAthletes = async (req, res, next) => {
+  try {
+    const list = await Athlete.find()
+      .sort([['lastName', 'ascending'], ['firstName', 'ascending']])
+    res.status(200).json(list.map(doc => util.fromMongoDb(doc)))
+  } catch (err) {
+    return next(err)
+  }
+}
+
+exports.getAthlete = async (req, res, next) => {
+  try {
+    // todo get GHIN index from ghin.com
+    const [athlete] = await Promise.all([Athlete.findById(req.params.id)])
+    if (athlete == null) {
+      return next(boom.notFound())
+    }
+    res.status(200).json(util.fromMongoDb(athlete))
+  } catch (err) {
+    return next(err)
+  }
+}
+exports.createAthlete = async (req, res, next) => {
+  try {
+    // todo index ghinNumber to avoid collection scan
+    const result = await Athlete.findOne({ ghinNumber: req.body.ghinNumber })
+    if (result && result.ghinNumber && result.ghinNumber === req.body.ghinNumber) {
```

Comment:
If you foresee this syntax being repeated across the code in the future, consider using lodash's get method to retrieve the value of a property which may or may not exist. Additionally, there is also a has method which returns a boolean whether the property exists or not.

Reply:
Thanks. I will take a look at lodash's get and has method. This syntax is cumbersome.

Reply:
I tested in repl and if (_.get(result, 'ghinNumber') === req.body.ghinNumber) is much cleaner. I haven't used lodash before but I can see it providing utility in a larger project.